packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/ComboboxFilter/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Combobox, ComboboxOption } from '@strapi/design-system/Combobox';
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalOverflowAuto = createGlobalStyle`
+  // Prevent scroll on combobox click when the page has an overflow
+  html, body {
+    overflow: auto;
+  }
```

Comment:
ℹ️ I put this here since the docs suggest that the styles will only be set when the component renders: https://styled-components.com/docs/api#createglobalstyle. Without the overflow: auto clicking the combobox in the popover causes a scroll when the page itself is scrollable. I also tried in the DS for this component: inputRef.current.focus({preventScroll: true}) and e.preventPropagation.